polkadot/node/subsystem-types/src/messages.rs

```diff
@@ -204,6 +204,50 @@ pub enum CandidateValidationMessage {
 	},
 }
 
+/// Extends primitives::PvfExecKind to have a separate value for duspute requests
+/// which is important for prioritization.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub enum PvfExecPriority {
```

Comment:
It is not clear to me what is the difference between this enum and PvfExecKind and why we can't extend PvfExecKind with a DisputeVariant, I assume, because ExecKind is used in the runtime as well ?

Reply:
Yes, ExecKind is a runtime param and there is no need to change it. I would say using it further in all that code was a confusion because we needed only 2 options.

Reply:
But I'll add a note here why we need it